tests/cases/fourslash/codeFixClassExtendsAbstractMethod.ts

```diff
@@ -0,0 +1,13 @@
+/// <reference path='fourslash.ts' />
+
+//// abstract class A {
+////    abstract f();
+//// }
+////
+//// class C extends A {[|
+//// |]}
+
+verify.rangeAfterCodeFix(`f(){
+    throw new Error('Method not Implemented');

```

Comment:
The new line characters are actually all removed in your `verify` functions. If you want to test the new lines as well, you may consider changing that.